App/features/photos/sagas.ts

```diff
@@ -14,9 +14,8 @@ import { ProcessingPhoto } from './models'
 
 function * toggleStorage(action: ActionType<typeof PreferencesActions.toggleStorageRequest>) {
   const { name } = action.payload
-  const storageOption = yield select(PreferencesSelectors.storage, name)
-  const storageStatus = !storageOption ? false : storageOption.status
```

Comment:
The storage options are always defined (they're all defined in the default state of the reducer), so simplified this a bit.